x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ccr/action/FollowInfoAction.java

```diff
@@ -189,7 +189,7 @@ public FollowParameters getParameters() {
                 remoteCluster = in.readString();
                 leaderIndex = in.readString();
                 status = Status.fromString(in.readString());
-                parameters = in.readOptionalWriteable(innerIn -> new FollowParameters(in));
+                parameters = in.readOptionalWriteable(FollowParameters::new);
```

Comment:
this change is unrelated, but it noticed that the FollowParameters was provided the `in` variable instead of `innerIn`. This is just a cosmetic change, as both variable point to the same instance.